projectgenerator/libili2db/ili2dbconfig.py

```diff
@@ -107,6 +116,7 @@ def __init__(self):
         self.dbhost = ''
         self.dbpwd = ''
         self.dbusr = ''
+        self.dbusesuperlogin = False
```

Comment:
can you use some `_` for readability in this variable name? The others are the way they are to be consistent with ili2db options (and this one is not an ili2db option if I am not wrong)